src/Maker/MakeResetPassword.php

```diff
@@ -125,7 +125,7 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
 
         $input->setArgument('from-email-address', $io->ask(
             'What email address will be used to send reset confirmations? e.g. mailer@your-domain.com',
-            null,
+            '',
             [Validator::class, 'validateEmailAddress']
         ));
```

Comment:
Allowing an empty value `''` would cause the email address validator to fail resulting in the question being repeated until a valid email address was provided.

Reply:
That IS the behavior we want (they should be forced to enter a value). So are you saying the patch is +1 for you?

Reply:
@weaverryan see my comment below. I think that better conveys what I was trying to say here.